packages/labs/analyzer/tsconfig.json

```diff
@@ -25,7 +25,7 @@
     "noFallthroughCasesInSwitch": true,
     "noImplicitOverride": true,
     "skipLibCheck": true,
-    "types": ["node"]
+    "types": ["node", "mocha"]
```

Comment:
I think ultimately the safest thing here would be to have separate tsconfigs:
1. The main library code, no node typings
2. The node test code and package-analyzer function, with node typings
3. _Maybe_ a third with browser typings if they were needed in browser tests.

Not having Node typings would have caught the place where we imported `path`.